uiowa/uiowa
# AMCS - fix for menu order

Resolves #6779 

# How to test
```
ddev blt ds --site=amcs.uiowa.edu && ddev drush @amcs.local uli /admin/structure/menu/manage/main
```

1. Verify that extra disabled links from Views are gone
2. Verify menu order matches #6779
